README.md

```diff
@@ -60,26 +60,15 @@ To make changes to Tilt, read the [developer guide](DEVELOPING.md).
 For bugs and feature requests, file an [issue](https://github.com/windmilleng/tilt/issues)
 or check out the [feature roadmap](ROADMAP.md).
 
-## Privacy
+## Telemetry and Privacy
+We're a small company trying to make Tilt awesomer. We can do this better if we understand which features people are using and which bugs people are running into. Tilt can send telemetry data to https://events.windmill.build if you opt-in. You can do this in the web UI or by running `tilt analytics opt in`. It really helps us improve your experience.
 
-This tool can send usage reports to https://events.windmill.build, to help us
-understand what features people use. We only report on which `tilt` commands
-run and how long they run for.
+The data is meant to be about your use of Tilt (which Tiltfile or Web UI features do you use), not collecting data about you or your project. It's possible that some of the data we collect could include snippets of data about your project (e.g. that you have a service named `deathray-backend` or an error message that includes the string it failed to parse). We try to avoid this, but you should probably not opt-in if you're working on a classified project.
```

Comment:
```suggestion
The data is meant to be about your use of Tilt (e.g., which Tiltfile or Web UI features do you use), not collecting data about you or your project. It's possible that some of the data we collect could include snippets of data about your project (e.g. that you have a service named `deathray-backend` or an error message that includes the string it failed to parse). We try to avoid this, but you should probably not opt in if you're working on a classified project.
```
(lol)